debian/rules

```diff
@@ -2,12 +2,14 @@
 # -*- makefile -*-
 
 export DH_OPTIONS
+PYTHON_BINARY := $(shell { command -v python3.4 || command -v python3.3 || command -v python2.7; } 2>/dev/null)
```

Comment:
I'm -1, this seems like a maintenance nightmare.

I'd be +1 to either `Depends: python3` or `Depends: python` except we need to support lucid